src/main/java/seedu/address/model/expenses/Expense.java

```diff
@@ -2,29 +2,51 @@
 
 import java.util.Date;
 
+import seedu.address.model.expenses.exceptions.ItemNotFoundException;
+
 /**
  * Represents a single spending on a single type of service.
  */
 public class Expense {
 
     private final double cost;
-    private final ExpenseType type;
+    private final String item;
     private final Date date;
```

Comment:
Might want to consider using `LocalDate` instead of `Date`. Refer to https://stackoverflow.com/questions/28730136/should-i-use-java-util-date-or-switch-to-java-time-localdate for pros/cons.